2024-08-07 18:18:58,672 INFO    StreamThr :23848 [internal.py:wandb_internal():86] W&B internal server running at pid: 23848, started at: 2024-08-07 18:18:58.672011
2024-08-07 18:18:58,673 DEBUG   HandlerThread:23848 [handler.py:handle_request():146] handle_request: status
2024-08-07 18:18:58,674 INFO    WriterThread:23848 [datastore.py:open_for_write():85] open: /home/rwkv/JL/GMM/llama/wandb/run-20240807_181858-iwj78x2c/run-iwj78x2c.wandb
2024-08-07 18:18:58,674 DEBUG   SenderThread:23848 [sender.py:send():382] send: header
2024-08-07 18:18:58,702 DEBUG   SenderThread:23848 [sender.py:send():382] send: run
2024-08-07 18:18:58,944 INFO    SenderThread:23848 [dir_watcher.py:__init__():211] watching files in: /home/rwkv/JL/GMM/llama/wandb/run-20240807_181858-iwj78x2c/files
2024-08-07 18:18:58,944 INFO    SenderThread:23848 [sender.py:_start_run_threads():1136] run started: iwj78x2c with start time 1723025938.671877
2024-08-07 18:18:58,951 DEBUG   HandlerThread:23848 [handler.py:handle_request():146] handle_request: check_version
2024-08-07 18:18:58,951 DEBUG   SenderThread:23848 [sender.py:send_request():409] send_request: check_version
2024-08-07 18:18:59,051 DEBUG   HandlerThread:23848 [handler.py:handle_request():146] handle_request: run_start
2024-08-07 18:18:59,067 DEBUG   HandlerThread:23848 [system_info.py:__init__():32] System info init
2024-08-07 18:18:59,067 DEBUG   HandlerThread:23848 [system_info.py:__init__():47] System info init done
2024-08-07 18:18:59,067 INFO    HandlerThread:23848 [system_monitor.py:start():194] Starting system monitor
2024-08-07 18:18:59,068 INFO    SystemMonitor:23848 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-08-07 18:18:59,068 INFO    HandlerThread:23848 [system_monitor.py:probe():214] Collecting system info
2024-08-07 18:18:59,069 INFO    SystemMonitor:23848 [interfaces.py:start():190] Started cpu monitoring
2024-08-07 18:18:59,070 INFO    SystemMonitor:23848 [interfaces.py:start():190] Started disk monitoring
2024-08-07 18:18:59,071 INFO    SystemMonitor:23848 [interfaces.py:start():190] Started gpu monitoring
2024-08-07 18:18:59,072 INFO    SystemMonitor:23848 [interfaces.py:start():190] Started memory monitoring
2024-08-07 18:18:59,074 INFO    SystemMonitor:23848 [interfaces.py:start():190] Started network monitoring
2024-08-07 18:18:59,086 DEBUG   HandlerThread:23848 [system_info.py:probe():196] Probing system
2024-08-07 18:18:59,087 DEBUG   HandlerThread:23848 [system_info.py:_probe_git():181] Probing git
2024-08-07 18:18:59,091 DEBUG   HandlerThread:23848 [system_info.py:_probe_git():189] Probing git done
2024-08-07 18:18:59,091 DEBUG   HandlerThread:23848 [system_info.py:probe():244] Probing system done
2024-08-07 18:18:59,091 DEBUG   HandlerThread:23848 [system_monitor.py:probe():223] {'os': 'Linux-6.5.0-44-generic-x86_64-with-glibc2.35', 'python': '3.10.13', 'heartbeatAt': '2024-08-07T10:18:59.086664', 'startedAt': '2024-08-07T10:18:58.668066', 'docker': None, 'cuda': None, 'args': ('--local_rank=0', '--deepspeed', 'configs/ds_config_zero2_no_offload.json', '--model_name_or_path', '/home/rwkv/JL/model/llama2-7b', '--data_path', '/home/rwkv/JL/model/MetaMathQA', '--dataset_field', 'query', 'response', '--dataset_split', 'train[:100000]', '--output_dir', '/home/rwkv/JL/out_model/llama-test', '--num_train_epochs', '1', '--model_max_length', '512', '--per_device_train_batch_size', '1', '--gradient_accumulation_steps', '32', '--save_strategy', 'steps', '--save_steps', '4000', '--save_total_limit', '2', '--learning_rate', '2e-5', '--weight_decay', '0.', '--warmup_ratio', '0.', '--logging_steps', '1', '--lr_scheduler_type', 'cosine', '--report_to', 'wandb', '--merge', 'True'), 'state': 'running', 'program': '/home/rwkv/JL/GMM/llama/gmm.py', 'codePathLocal': 'gmm.py', 'codePath': 'llama/gmm.py', 'git': {'remote': 'https://github.com/JL-er/GMM.git', 'commit': '74dce7f004b5b16bfd24c8ca22e737aee6ff0769'}, 'email': '997529190@qq.com', 'root': '/home/rwkv/JL/GMM', 'host': 'rwkv-Z790M-AORUS-ELITE-AX', 'username': 'rwkv', 'executable': '/home/rwkv/miniconda3/bin/python', 'cpu_count': 20, 'cpu_count_logical': 28, 'cpu_freq': {'current': 998.1528571428571, 'min': 800.0, 'max': 5000.0}, 'cpu_freq_per_core': [{'current': 1328.61, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 1683.881, 'min': 800.0, 'max': 5500.0}, {'current': 1436.659, 'min': 800.0, 'max': 5500.0}, {'current': 1340.011, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 1411.651, 'min': 800.0, 'max': 5500.0}, {'current': 1179.858, 'min': 800.0, 'max': 5500.0}, {'current': 1474.329, 'min': 800.0, 'max': 5600.0}, {'current': 1450.303, 'min': 800.0, 'max': 5600.0}, {'current': 1557.986, 'min': 800.0, 'max': 5600.0}, {'current': 1437.58, 'min': 800.0, 'max': 5600.0}, {'current': 1464.174, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 1250.821, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 1189.755, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}], 'disk': {'/': {'total': 916.2753829956055, 'used': 474.6733627319336}}, 'gpu': 'NVIDIA GeForce RTX 4090', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 4090', 'memory_total': 25757220864}], 'memory': {'total': 62.55397415161133}}
2024-08-07 18:18:59,091 INFO    HandlerThread:23848 [system_monitor.py:probe():224] Finished collecting system info
2024-08-07 18:18:59,091 INFO    HandlerThread:23848 [system_monitor.py:probe():227] Publishing system info
2024-08-07 18:18:59,091 DEBUG   HandlerThread:23848 [system_info.py:_save_pip():52] Saving list of pip packages installed into the current environment
2024-08-07 18:18:59,091 DEBUG   HandlerThread:23848 [system_info.py:_save_pip():68] Saving pip packages done
2024-08-07 18:18:59,091 DEBUG   HandlerThread:23848 [system_info.py:_save_conda():75] Saving list of conda packages installed into the current environment
2024-08-07 18:18:59,944 INFO    Thread-12 :23848 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_181858-iwj78x2c/files/requirements.txt
2024-08-07 18:18:59,944 INFO    Thread-12 :23848 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_181858-iwj78x2c/files/conda-environment.yaml
2024-08-07 18:19:00,801 DEBUG   HandlerThread:23848 [system_info.py:_save_conda():87] Saving conda packages done
2024-08-07 18:19:00,801 DEBUG   HandlerThread:23848 [system_info.py:_save_code():90] Saving code
2024-08-07 18:19:00,802 DEBUG   HandlerThread:23848 [system_info.py:_save_code():111] Saving code done
2024-08-07 18:19:00,802 DEBUG   HandlerThread:23848 [system_info.py:_save_patches():128] Saving git patches
2024-08-07 18:19:00,843 DEBUG   HandlerThread:23848 [system_info.py:_save_patches():170] Saving git patches done
2024-08-07 18:19:00,844 INFO    HandlerThread:23848 [system_monitor.py:probe():229] Finished publishing system info
2024-08-07 18:19:00,847 DEBUG   SenderThread:23848 [sender.py:send():382] send: files
2024-08-07 18:19:00,847 INFO    SenderThread:23848 [sender.py:_save_file():1392] saving file wandb-metadata.json with policy now
2024-08-07 18:19:00,848 INFO    SenderThread:23848 [sender.py:_save_file():1392] saving file code/llama/gmm.py with policy now
2024-08-07 18:19:00,848 INFO    SenderThread:23848 [sender.py:_save_file():1392] saving file diff.patch with policy now
2024-08-07 18:19:00,856 DEBUG   HandlerThread:23848 [handler.py:handle_request():146] handle_request: stop_status
2024-08-07 18:19:00,857 DEBUG   SenderThread:23848 [sender.py:send_request():409] send_request: stop_status
2024-08-07 18:19:00,861 DEBUG   HandlerThread:23848 [handler.py:handle_request():146] handle_request: internal_messages
2024-08-07 18:19:00,944 INFO    Thread-12 :23848 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240807_181858-iwj78x2c/files/conda-environment.yaml
2024-08-07 18:19:00,944 INFO    Thread-12 :23848 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_181858-iwj78x2c/files/wandb-metadata.json
2024-08-07 18:19:00,944 INFO    Thread-12 :23848 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_181858-iwj78x2c/files/code/llama/gmm.py
2024-08-07 18:19:00,944 INFO    Thread-12 :23848 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_181858-iwj78x2c/files/diff.patch
2024-08-07 18:19:00,944 INFO    Thread-12 :23848 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_181858-iwj78x2c/files/code/llama
2024-08-07 18:19:00,944 INFO    Thread-12 :23848 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_181858-iwj78x2c/files/code
2024-08-07 18:19:01,093 ERROR   wandb-upload_0:23848 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/iwj78x2c/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101900Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=733522a5f2468f139e3fe8ef681d68450677958f7bb6ae7d11bc15f8d466783a: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/iwj78x2c/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101900Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=733522a5f2468f139e3fe8ef681d68450677958f7bb6ae7d11bc15f8d466783a
2024-08-07 18:19:01,093 ERROR   wandb-upload_0:23848 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5218', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:19:01,093 ERROR   wandb-upload_0:23848 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/iwj78x2c/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/iwj78x2c/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E96AF6E50BC807</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:19:01,105 DEBUG   SenderThread:23848 [sender.py:send():382] send: telemetry
2024-08-07 18:19:01,105 DEBUG   SenderThread:23848 [sender.py:send():382] send: config
2024-08-07 18:19:01,106 DEBUG   SenderThread:23848 [sender.py:send():382] send: metric
2024-08-07 18:19:01,106 DEBUG   SenderThread:23848 [sender.py:send():382] send: telemetry
2024-08-07 18:19:01,106 DEBUG   SenderThread:23848 [sender.py:send():382] send: metric
2024-08-07 18:19:01,106 WARNING SenderThread:23848 [sender.py:send_metric():1343] Seen metric with glob (shouldn't happen)
2024-08-07 18:19:01,106 DEBUG   SenderThread:23848 [sender.py:send():382] send: telemetry
2024-08-07 18:19:01,106 DEBUG   SenderThread:23848 [sender.py:send():382] send: telemetry
2024-08-07 18:19:01,106 DEBUG   SenderThread:23848 [sender.py:send():382] send: config
2024-08-07 18:19:01,176 DEBUG   SenderThread:23848 [sender.py:send():382] send: exit
2024-08-07 18:19:01,177 INFO    SenderThread:23848 [sender.py:send_exit():589] handling exit code: 1
2024-08-07 18:19:01,177 INFO    SenderThread:23848 [sender.py:send_exit():591] handling runtime: 2
2024-08-07 18:19:01,179 INFO    SenderThread:23848 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-07 18:19:01,179 INFO    SenderThread:23848 [sender.py:send_exit():597] send defer
2024-08-07 18:19:01,180 DEBUG   HandlerThread:23848 [handler.py:handle_request():146] handle_request: defer
2024-08-07 18:19:01,180 INFO    HandlerThread:23848 [handler.py:handle_request_defer():172] handle defer: 0
2024-08-07 18:19:01,181 DEBUG   SenderThread:23848 [sender.py:send_request():409] send_request: defer
2024-08-07 18:19:01,181 INFO    SenderThread:23848 [sender.py:send_request_defer():613] handle sender defer: 0
2024-08-07 18:19:01,181 INFO    SenderThread:23848 [sender.py:transition_state():617] send defer: 1
2024-08-07 18:19:01,181 DEBUG   HandlerThread:23848 [handler.py:handle_request():146] handle_request: defer
2024-08-07 18:19:01,181 INFO    HandlerThread:23848 [handler.py:handle_request_defer():172] handle defer: 1
2024-08-07 18:19:01,181 DEBUG   SenderThread:23848 [sender.py:send_request():409] send_request: defer
2024-08-07 18:19:01,181 INFO    SenderThread:23848 [sender.py:send_request_defer():613] handle sender defer: 1
2024-08-07 18:19:01,181 INFO    SenderThread:23848 [sender.py:transition_state():617] send defer: 2
2024-08-07 18:19:01,182 DEBUG   HandlerThread:23848 [handler.py:handle_request():146] handle_request: defer
2024-08-07 18:19:01,182 INFO    HandlerThread:23848 [handler.py:handle_request_defer():172] handle defer: 2
2024-08-07 18:19:01,182 INFO    HandlerThread:23848 [system_monitor.py:finish():203] Stopping system monitor
2024-08-07 18:19:01,182 DEBUG   SystemMonitor:23848 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-08-07 18:19:01,182 DEBUG   SystemMonitor:23848 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2024-08-07 18:19:01,182 DEBUG   SystemMonitor:23848 [system_monitor.py:_start():183] Publishing last batch of metrics
2024-08-07 18:19:01,183 INFO    HandlerThread:23848 [interfaces.py:finish():202] Joined cpu monitor
2024-08-07 18:19:01,184 INFO    HandlerThread:23848 [interfaces.py:finish():202] Joined disk monitor
2024-08-07 18:19:01,207 ERROR   wandb-upload_1:23848 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/iwj78x2c/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101901Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=f2d66f3438ce351a945dba86be5f4238a3a2eb1cc581b3995bfb3d6719fa4760: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/iwj78x2c/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101901Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=f2d66f3438ce351a945dba86be5f4238a3a2eb1cc581b3995bfb3d6719fa4760
2024-08-07 18:19:01,207 ERROR   wandb-upload_1:23848 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:19:01,207 ERROR   wandb-upload_1:23848 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/iwj78x2c/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/iwj78x2c/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E96AF6E9434626</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:19:01,218 INFO    HandlerThread:23848 [interfaces.py:finish():202] Joined gpu monitor
2024-08-07 18:19:01,218 INFO    HandlerThread:23848 [interfaces.py:finish():202] Joined memory monitor
2024-08-07 18:19:01,218 INFO    HandlerThread:23848 [interfaces.py:finish():202] Joined network monitor
2024-08-07 18:19:01,218 DEBUG   SenderThread:23848 [sender.py:send_request():409] send_request: defer
2024-08-07 18:19:01,218 INFO    SenderThread:23848 [sender.py:send_request_defer():613] handle sender defer: 2
2024-08-07 18:19:01,218 INFO    SenderThread:23848 [sender.py:transition_state():617] send defer: 3
2024-08-07 18:19:01,218 DEBUG   SenderThread:23848 [sender.py:send():382] send: stats
2024-08-07 18:19:01,218 DEBUG   HandlerThread:23848 [handler.py:handle_request():146] handle_request: defer
2024-08-07 18:19:01,218 INFO    HandlerThread:23848 [handler.py:handle_request_defer():172] handle defer: 3
2024-08-07 18:19:01,219 DEBUG   SenderThread:23848 [sender.py:send_request():409] send_request: defer
2024-08-07 18:19:01,219 INFO    SenderThread:23848 [sender.py:send_request_defer():613] handle sender defer: 3
2024-08-07 18:19:01,219 INFO    SenderThread:23848 [sender.py:transition_state():617] send defer: 4
2024-08-07 18:19:01,219 DEBUG   HandlerThread:23848 [handler.py:handle_request():146] handle_request: defer
2024-08-07 18:19:01,219 INFO    HandlerThread:23848 [handler.py:handle_request_defer():172] handle defer: 4
2024-08-07 18:19:01,219 DEBUG   SenderThread:23848 [sender.py:send_request():409] send_request: defer
2024-08-07 18:19:01,219 INFO    SenderThread:23848 [sender.py:send_request_defer():613] handle sender defer: 4
2024-08-07 18:19:01,219 INFO    SenderThread:23848 [sender.py:transition_state():617] send defer: 5
2024-08-07 18:19:01,219 DEBUG   HandlerThread:23848 [handler.py:handle_request():146] handle_request: defer
2024-08-07 18:19:01,219 INFO    HandlerThread:23848 [handler.py:handle_request_defer():172] handle defer: 5
2024-08-07 18:19:01,219 DEBUG   SenderThread:23848 [sender.py:send():382] send: summary
2024-08-07 18:19:01,219 INFO    SenderThread:23848 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-07 18:19:01,219 DEBUG   SenderThread:23848 [sender.py:send_request():409] send_request: defer
2024-08-07 18:19:01,219 INFO    SenderThread:23848 [sender.py:send_request_defer():613] handle sender defer: 5
2024-08-07 18:19:01,219 INFO    SenderThread:23848 [sender.py:transition_state():617] send defer: 6
2024-08-07 18:19:01,219 DEBUG   HandlerThread:23848 [handler.py:handle_request():146] handle_request: defer
2024-08-07 18:19:01,219 INFO    HandlerThread:23848 [handler.py:handle_request_defer():172] handle defer: 6
2024-08-07 18:19:01,219 DEBUG   SenderThread:23848 [sender.py:send_request():409] send_request: defer
2024-08-07 18:19:01,219 INFO    SenderThread:23848 [sender.py:send_request_defer():613] handle sender defer: 6
2024-08-07 18:19:01,220 DEBUG   HandlerThread:23848 [handler.py:handle_request():146] handle_request: status_report
2024-08-07 18:19:01,341 ERROR   wandb-upload_2:23848 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/iwj78x2c/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101901Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=93657a42c4ebba82cce9d7621430ff11a3c0199341c35b4d6c94e5bd4d887581: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/iwj78x2c/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101901Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=93657a42c4ebba82cce9d7621430ff11a3c0199341c35b4d6c94e5bd4d887581
2024-08-07 18:19:01,341 ERROR   wandb-upload_2:23848 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '287189', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:19:01,341 ERROR   wandb-upload_2:23848 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/iwj78x2c/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/iwj78x2c/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E96AF6E83A8EFA</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:19:01,465 INFO    SenderThread:23848 [sender.py:transition_state():617] send defer: 7
2024-08-07 18:19:01,465 DEBUG   HandlerThread:23848 [handler.py:handle_request():146] handle_request: defer
2024-08-07 18:19:01,465 INFO    HandlerThread:23848 [handler.py:handle_request_defer():172] handle defer: 7
2024-08-07 18:19:01,466 DEBUG   SenderThread:23848 [sender.py:send_request():409] send_request: defer
2024-08-07 18:19:01,466 INFO    SenderThread:23848 [sender.py:send_request_defer():613] handle sender defer: 7
2024-08-07 18:19:01,945 INFO    Thread-12 :23848 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240807_181858-iwj78x2c/files/config.yaml
2024-08-07 18:19:01,945 INFO    Thread-12 :23848 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_181858-iwj78x2c/files/wandb-summary.json
2024-08-07 18:19:01,946 INFO    Thread-12 :23848 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_181858-iwj78x2c/files/output.log
2024-08-07 18:19:02,171 DEBUG   HandlerThread:23848 [handler.py:handle_request():146] handle_request: poll_exit
2024-08-07 18:19:02,184 ERROR   wandb-upload_0:23848 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/iwj78x2c/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101900Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=733522a5f2468f139e3fe8ef681d68450677958f7bb6ae7d11bc15f8d466783a: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/iwj78x2c/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101900Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=733522a5f2468f139e3fe8ef681d68450677958f7bb6ae7d11bc15f8d466783a
2024-08-07 18:19:02,184 ERROR   wandb-upload_0:23848 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5218', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:19:02,184 ERROR   wandb-upload_0:23848 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/iwj78x2c/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/iwj78x2c/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E96AF72603F6A1</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:19:02,377 ERROR   wandb-upload_1:23848 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/iwj78x2c/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101901Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=f2d66f3438ce351a945dba86be5f4238a3a2eb1cc581b3995bfb3d6719fa4760: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/iwj78x2c/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101901Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=f2d66f3438ce351a945dba86be5f4238a3a2eb1cc581b3995bfb3d6719fa4760
2024-08-07 18:19:02,377 ERROR   wandb-upload_1:23848 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:19:02,377 ERROR   wandb-upload_1:23848 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/iwj78x2c/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/iwj78x2c/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E96AF73179347E</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:19:02,377 INFO    wandb-upload_1:23848 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/iwj78x2c/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101901Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=f2d66f3438ce351a945dba86be5f4238a3a2eb1cc581b3995bfb3d6719fa4760

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2785, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
wandb.sdk.lib.retry.TransientError: None
2024-08-07 18:19:02,480 ERROR   wandb-upload_2:23848 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/iwj78x2c/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101901Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=93657a42c4ebba82cce9d7621430ff11a3c0199341c35b4d6c94e5bd4d887581: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/iwj78x2c/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101901Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=93657a42c4ebba82cce9d7621430ff11a3c0199341c35b4d6c94e5bd4d887581
2024-08-07 18:19:02,480 ERROR   wandb-upload_2:23848 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '287189', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:19:02,480 ERROR   wandb-upload_2:23848 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/iwj78x2c/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/iwj78x2c/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E96AF7357DB7F8</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:19:03,120 INFO    SenderThread:23848 [sender.py:transition_state():617] send defer: 8
2024-08-07 18:19:03,120 DEBUG   SenderThread:23848 [sender.py:send_request():409] send_request: poll_exit
2024-08-07 18:19:03,120 DEBUG   HandlerThread:23848 [handler.py:handle_request():146] handle_request: defer
2024-08-07 18:19:03,120 INFO    HandlerThread:23848 [handler.py:handle_request_defer():172] handle defer: 8
2024-08-07 18:19:03,121 DEBUG   SenderThread:23848 [sender.py:send_request():409] send_request: defer
2024-08-07 18:19:03,121 INFO    SenderThread:23848 [sender.py:send_request_defer():613] handle sender defer: 8
2024-08-07 18:19:03,121 INFO    SenderThread:23848 [job_builder.py:build():298] Attempting to build job artifact
2024-08-07 18:19:03,121 INFO    SenderThread:23848 [job_builder.py:_get_source_type():428] is repo sourced job
2024-08-07 18:19:03,135 INFO    SenderThread:23848 [job_builder.py:build():404] adding wandb-job metadata file
2024-08-07 18:19:03,146 INFO    SenderThread:23848 [sender.py:transition_state():617] send defer: 9
2024-08-07 18:19:03,147 DEBUG   HandlerThread:23848 [handler.py:handle_request():146] handle_request: defer
2024-08-07 18:19:03,147 INFO    HandlerThread:23848 [handler.py:handle_request_defer():172] handle defer: 9
2024-08-07 18:19:03,147 DEBUG   SenderThread:23848 [sender.py:send():382] send: artifact
2024-08-07 18:19:03,172 DEBUG   HandlerThread:23848 [handler.py:handle_request():146] handle_request: poll_exit
2024-08-07 18:19:03,882 ERROR   wandb-upload_3:23848 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/508/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101903Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=f202442ad4d802e508b2e7318ca567dfe6300ee2174e8cf89cd0453b8bd81481: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/508/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101903Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=f202442ad4d802e508b2e7318ca567dfe6300ee2174e8cf89cd0453b8bd81481
2024-08-07 18:19:03,882 ERROR   wandb-upload_3:23848 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-MD5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:19:03,882 ERROR   wandb-upload_3:23848 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/508/c2df8f0a6cb4e05226b857b6cdc53f4d</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/508/c2df8f0a6cb4e05226b857b6cdc53f4d</Resource><Region>wandb-local</Region><RequestId>17E96AF78B353732</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:19:03,946 INFO    Thread-12 :23848 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240807_181858-iwj78x2c/files/output.log
2024-08-07 18:19:03,962 ERROR   wandb-upload_4:23848 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/508/99df597bfbd8f48d94032ad571ab30ec?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101903Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=ca3d8a112997e8ac32f9656d85591f470b24a570bdf68f2c20809176496ee448: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/508/99df597bfbd8f48d94032ad571ab30ec?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101903Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=ca3d8a112997e8ac32f9656d85591f470b24a570bdf68f2c20809176496ee448
2024-08-07 18:19:03,962 ERROR   wandb-upload_4:23848 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'md9Ze/vY9I2UAyrVcasw7A==', 'Content-MD5': 'md9Ze/vY9I2UAyrVcasw7A==', 'Content-Type': 'application/json', 'Content-Length': '25169', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:19:03,962 ERROR   wandb-upload_4:23848 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/508/99df597bfbd8f48d94032ad571ab30ec</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/508/99df597bfbd8f48d94032ad571ab30ec</Resource><Region>wandb-local</Region><RequestId>17E96AF78B088B6D</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:19:04,669 ERROR   wandb-upload_0:23848 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/iwj78x2c/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101900Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=733522a5f2468f139e3fe8ef681d68450677958f7bb6ae7d11bc15f8d466783a: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/iwj78x2c/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101900Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=733522a5f2468f139e3fe8ef681d68450677958f7bb6ae7d11bc15f8d466783a
2024-08-07 18:19:04,669 ERROR   wandb-upload_0:23848 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5218', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:19:04,669 ERROR   wandb-upload_0:23848 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/iwj78x2c/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/iwj78x2c/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E96AF7BA16B152</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:19:04,758 ERROR   wandb-upload_2:23848 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/iwj78x2c/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101901Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=93657a42c4ebba82cce9d7621430ff11a3c0199341c35b4d6c94e5bd4d887581: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/iwj78x2c/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101901Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=93657a42c4ebba82cce9d7621430ff11a3c0199341c35b4d6c94e5bd4d887581
2024-08-07 18:19:04,758 ERROR   wandb-upload_2:23848 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '287189', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:19:04,758 ERROR   wandb-upload_2:23848 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/iwj78x2c/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/iwj78x2c/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E96AF7BDB8994D</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:19:04,791 ERROR   wandb-upload_1:23848 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/iwj78x2c/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101901Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=f2d66f3438ce351a945dba86be5f4238a3a2eb1cc581b3995bfb3d6719fa4760: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/iwj78x2c/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101901Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=f2d66f3438ce351a945dba86be5f4238a3a2eb1cc581b3995bfb3d6719fa4760
2024-08-07 18:19:04,792 ERROR   wandb-upload_1:23848 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:19:04,792 ERROR   wandb-upload_1:23848 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/iwj78x2c/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/iwj78x2c/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E96AF7C0BD932E</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:19:05,002 ERROR   wandb-upload_3:23848 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/508/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101903Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=f202442ad4d802e508b2e7318ca567dfe6300ee2174e8cf89cd0453b8bd81481: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/508/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101903Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=f202442ad4d802e508b2e7318ca567dfe6300ee2174e8cf89cd0453b8bd81481
2024-08-07 18:19:05,002 ERROR   wandb-upload_3:23848 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-MD5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:19:05,002 ERROR   wandb-upload_3:23848 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/508/c2df8f0a6cb4e05226b857b6cdc53f4d</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/508/c2df8f0a6cb4e05226b857b6cdc53f4d</Resource><Region>wandb-local</Region><RequestId>17E96AF7CDF19CE1</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:19:05,314 ERROR   wandb-upload_4:23848 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/508/99df597bfbd8f48d94032ad571ab30ec?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101903Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=ca3d8a112997e8ac32f9656d85591f470b24a570bdf68f2c20809176496ee448: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/508/99df597bfbd8f48d94032ad571ab30ec?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101903Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=ca3d8a112997e8ac32f9656d85591f470b24a570bdf68f2c20809176496ee448
2024-08-07 18:19:05,314 ERROR   wandb-upload_4:23848 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'md9Ze/vY9I2UAyrVcasw7A==', 'Content-MD5': 'md9Ze/vY9I2UAyrVcasw7A==', 'Content-Type': 'application/json', 'Content-Length': '25169', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:19:05,314 ERROR   wandb-upload_4:23848 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/508/99df597bfbd8f48d94032ad571ab30ec</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/508/99df597bfbd8f48d94032ad571ab30ec</Resource><Region>wandb-local</Region><RequestId>17E96AF7E09D9B70</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:19:07,560 ERROR   wandb-upload_3:23848 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/508/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101903Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=f202442ad4d802e508b2e7318ca567dfe6300ee2174e8cf89cd0453b8bd81481: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/508/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101903Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=f202442ad4d802e508b2e7318ca567dfe6300ee2174e8cf89cd0453b8bd81481
2024-08-07 18:19:07,561 ERROR   wandb-upload_3:23848 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-MD5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:19:07,561 ERROR   wandb-upload_3:23848 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/508/c2df8f0a6cb4e05226b857b6cdc53f4d</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/508/c2df8f0a6cb4e05226b857b6cdc53f4d</Resource><Region>wandb-local</Region><RequestId>17E96AF865EFCB1A</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:19:07,561 INFO    wandb-upload_3:23848 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/508/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101903Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=f202442ad4d802e508b2e7318ca567dfe6300ee2174e8cf89cd0453b8bd81481

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2785, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
wandb.sdk.lib.retry.TransientError: None
2024-08-07 18:19:07,872 ERROR   wandb-upload_4:23848 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/508/99df597bfbd8f48d94032ad571ab30ec?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101903Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=ca3d8a112997e8ac32f9656d85591f470b24a570bdf68f2c20809176496ee448: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/508/99df597bfbd8f48d94032ad571ab30ec?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101903Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=ca3d8a112997e8ac32f9656d85591f470b24a570bdf68f2c20809176496ee448
2024-08-07 18:19:07,872 ERROR   wandb-upload_4:23848 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'md9Ze/vY9I2UAyrVcasw7A==', 'Content-MD5': 'md9Ze/vY9I2UAyrVcasw7A==', 'Content-Type': 'application/json', 'Content-Length': '25169', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:19:07,872 ERROR   wandb-upload_4:23848 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/508/99df597bfbd8f48d94032ad571ab30ec</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/508/99df597bfbd8f48d94032ad571ab30ec</Resource><Region>wandb-local</Region><RequestId>17E96AF877337D57</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:19:07,872 INFO    wandb-upload_4:23848 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/508/99df597bfbd8f48d94032ad571ab30ec?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101903Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=ca3d8a112997e8ac32f9656d85591f470b24a570bdf68f2c20809176496ee448

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2785, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
wandb.sdk.lib.retry.TransientError: None
2024-08-07 18:19:08,175 DEBUG   HandlerThread:23848 [handler.py:handle_request():146] handle_request: keepalive
2024-08-07 18:19:08,949 ERROR   wandb-upload_0:23848 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/iwj78x2c/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101900Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=733522a5f2468f139e3fe8ef681d68450677958f7bb6ae7d11bc15f8d466783a: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/iwj78x2c/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101900Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=733522a5f2468f139e3fe8ef681d68450677958f7bb6ae7d11bc15f8d466783a
2024-08-07 18:19:08,950 ERROR   wandb-upload_0:23848 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5218', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:19:08,950 ERROR   wandb-upload_0:23848 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/iwj78x2c/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/iwj78x2c/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E96AF8B9191608</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:19:09,457 ERROR   wandb-upload_1:23848 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/iwj78x2c/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101901Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=f2d66f3438ce351a945dba86be5f4238a3a2eb1cc581b3995bfb3d6719fa4760: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/iwj78x2c/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101901Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=f2d66f3438ce351a945dba86be5f4238a3a2eb1cc581b3995bfb3d6719fa4760
2024-08-07 18:19:09,457 ERROR   wandb-upload_1:23848 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:19:09,457 ERROR   wandb-upload_1:23848 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/iwj78x2c/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/iwj78x2c/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E96AF8D6C1F64A</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:19:09,874 ERROR   wandb-upload_2:23848 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/iwj78x2c/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101901Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=93657a42c4ebba82cce9d7621430ff11a3c0199341c35b4d6c94e5bd4d887581: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/iwj78x2c/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101901Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=93657a42c4ebba82cce9d7621430ff11a3c0199341c35b4d6c94e5bd4d887581
2024-08-07 18:19:09,874 ERROR   wandb-upload_2:23848 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '287189', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:19:09,874 ERROR   wandb-upload_2:23848 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/iwj78x2c/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/iwj78x2c/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E96AF8E6401C1A</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:19:12,637 ERROR   wandb-upload_3:23848 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/508/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101903Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=f202442ad4d802e508b2e7318ca567dfe6300ee2174e8cf89cd0453b8bd81481: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/508/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101903Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=f202442ad4d802e508b2e7318ca567dfe6300ee2174e8cf89cd0453b8bd81481
2024-08-07 18:19:12,637 ERROR   wandb-upload_3:23848 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-MD5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:19:12,637 ERROR   wandb-upload_3:23848 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/508/c2df8f0a6cb4e05226b857b6cdc53f4d</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/508/c2df8f0a6cb4e05226b857b6cdc53f4d</Resource><Region>wandb-local</Region><RequestId>17E96AF98E9BD47B</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:19:12,797 ERROR   wandb-upload_4:23848 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/508/99df597bfbd8f48d94032ad571ab30ec?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101903Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=ca3d8a112997e8ac32f9656d85591f470b24a570bdf68f2c20809176496ee448: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/508/99df597bfbd8f48d94032ad571ab30ec?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T101903Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=ca3d8a112997e8ac32f9656d85591f470b24a570bdf68f2c20809176496ee448
2024-08-07 18:19:12,797 ERROR   wandb-upload_4:23848 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'md9Ze/vY9I2UAyrVcasw7A==', 'Content-MD5': 'md9Ze/vY9I2UAyrVcasw7A==', 'Content-Type': 'application/json', 'Content-Length': '25169', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 18:19:12,797 ERROR   wandb-upload_4:23848 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/508/99df597bfbd8f48d94032ad571ab30ec</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/508/99df597bfbd8f48d94032ad571ab30ec</Resource><Region>wandb-local</Region><RequestId>17E96AF99CBD4A7F</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 18:19:13,177 DEBUG   HandlerThread:23848 [handler.py:handle_request():146] handle_request: keepalive
